tables/ru-ru-g1.ctb

```diff
@@ -0,0 +1,664 @@
+# liblouis: Russian literary (grade 1) text translation table
+#
+#-index-name: Russian, grade 1
+#-display-name: Russian contracted braille
+#
+#+locale: ru
+#+type: literary
+#+dots: 6
+#+contraction: partial
```

Comment:
partial: is this correct?

Reply:
Yes. We also have a grade 2, but it seems to be outdated, although I'm note sure. I don't know if we will create the Russian grade 2 table.

Reply:
OK, cool.